packages/venia-ui/lib/components/Button/__stories__/button.js

```diff
@@ -0,0 +1,162 @@
+import React from 'react';
+
+import Button from '../button';
+
+import { storiesOf } from '@storybook/react';
+import { Edit2, Menu } from 'react-feather';
+
+import styleOverrides from './button.overrides.module.css';
+
+const stories = storiesOf('Components/Button', module);
+
+const SIZES = ['large', 'medium', 'small'];
+
+const ICON_DIMENSIONS = { width: '18px', height: '18px' };
+
+const ButtonStory = props => {
+    const { design } = props;
+
+    const clickHandler = () => {
+        console.log('Click');
+    };
+
+    const buttons = SIZES.map(size => {
+        const text = `${design} ${size}`;
+        return (
+            <>
+                <Button
+                    key={`${design} ${size}`}
+                    design={design}
+                    size={size}
+                    ariaLabel={`${design} ${size}`}
+                    onClick={clickHandler}
+                >
+                    {text}
+                </Button>
+                <br />
+                <br />
+            </>
+        );
+    });
+
+    const disabledButtons = SIZES.map(size => {
+        const text = `${design} ${size}`;
+        return (
+            <>
+                <Button
+                    key={`${design} ${size}`}
+                    design={design}
+                    size={size}
+                    ariaLabel={`${design} ${size}`}
+                    disabled={true}
+                    onClick={clickHandler}
+                >
+                    {text}
+                </Button>
+                <br />
+                <br />
+            </>
+        );
+    });
+
+    const icon = <Edit2 {...ICON_DIMENSIONS} />;
+    const enabledText = 'Enabled';
+    const disabledText = 'Disabled';
+    const iconsText = 'Icons';
+    return (
+        <>
+            <h2>{enabledText}</h2>
+            {buttons}
+            <h2>{disabledText}</h2>
+            {disabledButtons}
+            <h2>{iconsText}</h2>
+            <div>
+                <Button
+                    design={design}
+                    ariaLabel={`${design} Left Icon`}
```

Comment:
Wondering if we need to add support for translations in Storybook, or could add this as a part of the Storybook Improvement epic.

Reply:
would be better to put it in the improvement epic